### DISCUSSION TITLE
Ideas

### DISCUSSION THREAD

**Discussion by changcheng967** [changcheng967/Kata_speed#112] [Ideas]
Here to upload ideas

**Comment by changcheng967** [comment]
I want to create a script that can detect level on OGS and you can change the level in KataGo human example to make it more humalize and make it more friendly

**Reply by changcheng22023** [reply]
Agree

**Comment by changcheng967** [comment]
In Computer Go Server, Kata-speed has achieved a visit of 2180000!
